data/2016/zoneData/Z1_taskProps.json

```diff
@@ -19509,5 +19508,7635 @@
         "id": 140559794
       }
     ]
+  },
+  {
+    "actor_file": "Common_Props_BarbedWire_Posts.adr",
+    "modelId": 43,
+    "renderDistance": 50,
+    "instances": [
+      {
+        "position": [
+          -3479.26318359375,
```

Comment:
these spawned barbedWire aren't destroyed when "killed" the model still there but make no damage

Reply:
I remember this problem from #1765, this is also an issue with punji sticks and has been since they were implemented. 

A fix for this could be to check if a barbed wire is a world spawn or a player placed it. From this check we can make it so if it's a world spawned, then to not allow it to be destroyed, and if it's a player spawn then to allow anyone to destroy it. If I recall correctly I think this is how DBG did it also 😂😂